src/librustc_codegen_llvm/back/lto.rs

```diff
@@ -634,9 +634,7 @@ pub(crate) fn run_pass_manager(
             llvm::LLVMRustAddPass(pm, pass.unwrap());
         }
 
-        cgcx.prof
-            .extra_verbose_generic_activity("LTO_passes")
```

Comment:
Any reason to remove this timer? Just curious.

Reply:
It's replaced by `LLVM_lto_optimize` above.